### DISCUSSION TITLE
google pubsub sink

### DISCUSSION THREAD

**Discussion by docteurklein** [risingwavelabs/risingwave#15237] [Ideas]
Hey there! 

Thanks again for the amazing work, love it all!

Any plan to introduce pubsub as a **sink**?

**Comment by yingjunwu** [comment]
Hi @docteurklein - right now we don't really have pub/sub sink, but based on our internal discussion, it should be a straigtforward implementation which won't take too much time. I wonder if you could open an issue and describe your requirements? We will try to add it into the next release.

btw, we do not monitor GitHub Discussions actively. The best way to find support is still through our Slack community: risingwave.com/slack. Thanks :-)

**Comment by docteurklein** [comment]
Thanks, good to know. I'll open an issue then and avoid GH discussions :)
